### PR TITLE
1.32.0

### PR DESCRIPTION
**Pull request by denobot**
Bumped versions for 1.32.0

Please ensure:
- [x] Target branch is correct
- [x] Crate versions are bumped correctly
- [x] deno_std version is incremented in the code (see `cli/deno_std.rs`)
- [x] Releases.md is updated correctly

To make edits to this PR:
```shell
git fetch upstream release_1_32.0 && git checkout -b release_1_32.0 upstream/release_1_32.0
```

cc @mmastrac

### CODE REVIEW

File: Releases.md [hunk 1]
```diff
@@ -6,6 +6,44 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at:
 https://github.com/denoland/deno_install
 
+### 1.32.0 / 2023.03.22
+
+- BREAKING(ext/fs): FileInfo.dev is defined on Windows (#18237)
+- BREAKING(unstable): remove WebGPU (#18094)
+- Revert "refactor: rename Deno.openKv() to Deno.kv() (#18349)" (#18362)
+- feat(cli): --ext parameter for run, compile, and bundle (#17172)
```
Comment (dsherret) [inline Releases.md:14]:
```suggestion
- feat(cli): --ext parameter for run, compile, and bundle (#17172)
```

File: Releases.md [hunk 2]
```diff
@@ -6,6 +6,44 @@ https://github.com/denoland/deno/releases
...
+- feat(compile): Add support for web workers in standalone mode (#17657)
+- feat(compile): Enable multiple roots for a standalone module graph (#17663)
+- feat(core): deno_core::extension! macro to simplify extension registration
+  (#18210)
+- feat(ext/kv): key-value store (#18232)
+- feat(ext/net): Add multicasting APIs to DatagramConn (#10706) (#17811)
+- feat(ext/url): URLSearchParams.size (#17884)
+- feat(fs): support FileInfo.dev on Windows (#18073)
```
Comment (bartlomieju) [inline Releases.md:22]: Let's remove this one as it's mentioned as the first `BREAKING(ext/fs)`

File: Releases.md [hunk 3]
```diff
@@ -6,6 +6,44 @@ https://github.com/denoland/deno/releases
...
+- fix(cli/integration): clippy lints (#18248)
+- fix(ext/kv): don't request permissions for ":memory:" (#18346)
+- fix(ext/kv): reverse mapping between `AnyValue::Bool` and `KeyPart::Bool`
+  (#18365)
+- fix(ext/node): add createDecipheriv (#18245)
+- fix(ext/node): resource leak in createHmac (#18229)
+- fix(ext/node): use Deno.Command from `ext:runtime` (#18289)
+- fix(repl): Hide indexable properties in tab completion (#18141)
+- fix(runtime): Extract error code for all OS error variants (#17958)
+- fix(test_util): parse strace usecs/call in total line if present (#18241)
```
Comment (bartlomieju) [inline Releases.md:36]: This is internal code, let's skip it

File: Releases.md (hunk 2 above)
Comment (dsherret) [reply Releases.md:21]: I feel like we should remove the breaking one. It wasn't really breaking since it was a fix to a breaking.

Comment (bartlomieju) [reply Releases.md:21]: It's a breaking change because the typing changed from `null | number` to `number`

Comment (dsherret) [reply Releases.md]: It was `number | null` then `number` by accident (breaking) and now the fix is to restore the default behaviour, which I wouldn't really consider a breaking change, but just a fix.

Comment (bartlomieju) [reply Releases.md]: > It was number | null then number by accident (breaking) and now the fix is to restore the default behaviour, which I wouldn't really consider a breaking change, but just a fix.

No, it's still been changed for the `FileInfo.dev` fields, the remaining fields were reverted

**Review by mmastrac — APPROVED** [review]

**Pull request merged**